src/components/editor/property/PropertyTemplateOutline.jsx

```diff
@@ -88,7 +88,7 @@ export class PropertyTemplateOutline extends Component {
 
   render() {
     return (
-      <div className="rtOutline">
+      <div className="rtOutline" propLabel={this.props.propertyTemplate.propertyLabel}>
```

Comment:
adding this propLabel attribute makes CSS selectors for integration tests a lot more clear ...